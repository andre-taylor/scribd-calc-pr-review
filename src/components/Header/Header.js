import React from "react"
import PropTypes from "prop-types"

import styles from "./Header.module.css"

const Header = ({ children }) => {
  return <h2 className={styles.tradeMark}>{children}</h2>
}

export default Header

Header.propTypes = {
  children: PropTypes.node.isRequired
}
