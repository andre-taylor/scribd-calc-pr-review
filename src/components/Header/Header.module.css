.tradeMark {
  font-size: 11px;
  color: #a0a0a0;
}
