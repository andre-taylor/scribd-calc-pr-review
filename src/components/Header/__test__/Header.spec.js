import React from "react"
import renderer from "react-test-renderer"

import Header from "../Header"

describe("<Header />", () => {
  it("renders correctly with required props", () => {
    const component = renderer.create(
      <Header>
        basic calculator<sup>&trade;</sup>
      </Header>
    )
    expect(component).toMatchSnapshot()
  })
})
