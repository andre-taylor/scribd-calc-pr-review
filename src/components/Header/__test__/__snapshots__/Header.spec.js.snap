// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly with required props 1`] = `
<h2
  className="tradeMark"
>
  basic calculator
  <sup>
    â„¢
  </sup>
</h2>
`;
